doc/source/plans.rst

```diff
@@ -649,6 +649,4 @@ These are useful utilities for defining custom plans and plan preprocessors.
    :nosignatures:
 
     broadcast_msg
-    repeater
```

Comment:
I only recently listed these here in the first place (since the most recent tag). Now, since `caching_repeater` is being deprecated and `repeater` is inferior to `repeat`, I'm rolling back my decision to list them here.